vm/runtime.ts

```diff
@@ -1700,7 +1700,7 @@ module J2ME {
     }
 
     // Don't compile methods that are too large.
-    if (methodInfo.codeAttribute.code.length > 2000 && !config.forceRuntimeCompilation) {
+    if (methodInfo.codeAttribute.code.length > 4000 && !config.forceRuntimeCompilation) {
```

Comment:
We should investigate how increasing this affects memory usage (with https://github.com/mozilla/j2me.js/wiki/Tracking-Memory-Usage).